dovpanda/base.py

```diff
@@ -9,6 +9,7 @@
 import pandas
 
 PANDAS_DIR = str(pathlib.Path(pandas.__file__).parent.absolute())
+MAX_CSV_SIZE = 500000
```

Comment:
is this bytes? kbytes? answer that in an inline comment

Reply:
Also updated the amount.